build.fsx

```diff
@@ -311,7 +311,19 @@ Target.create "WebApp.Watch" (fun _ ->
 )
 
 Target.create "WebApp.Publish" (fun _ ->
-    Yarn.exec "run gh-pages --dist web-app/output" id
+    if (isAppveyor) then
+        Yarn.exec (sprintf "version --new-version %s --no-git-tag-version" version) id
+
+        let repoName = Environment.environVar "appveyor_repo_name"
+        let repoBranch = Environment.environVar "appveyor_repo_branch"
+        let prHeadRepoName = Environment.environVar "APPVEYOR_PULL_REQUEST_HEAD_REPO_NAME"
+
+        if repoName = "fable-compiler/ts2fable" && repoBranch = "master" && String.isNullOrEmpty prHeadRepoName then
+            let line = sprintf "//registry.npmjs.org/:_authToken=%s\n" <| Environment.environVar "npmauthtoken"
+            let npmrc = (Fake.SystemHelper.Environment.GetFolderPath Fake.SystemHelper.Environment.UserProfile)</>".npmrc"
+            File.writeNew npmrc [line]
+            npm "whoami"
+            Yarn.exec "run gh-pages --dist web-app/output" id
```

Comment:
Just run  `yarn run gh-pages --dist web-app/output`
Is there others thinks needed to set to publish to github pages

Reply:
Running the target `WebApp.Publish` should be enough.
However, it will probably need the user/password for the github account :)